src/main/java/me/rigamortis/seppuku/impl/gui/hud/component/PlayerPreviewComponent.java

```diff
@@ -0,0 +1,68 @@
+package me.rigamortis.seppuku.impl.gui.hud.component;
+
+import me.rigamortis.seppuku.api.gui.hud.component.DraggableHudComponent;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiChat;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.OpenGlHelper;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.entity.player.EntityPlayer;
+import org.lwjgl.opengl.GL11;
+
+/**
+ * @author cookiedragon234 10/Dec/2019
+ */
+public class PlayerPreviewComponent extends DraggableHudComponent {
+	
+	private static final int INVENTORY_WIDTH = 49;
+	private static final int INVENTORY_HEIGHT = 70;
+	
+	public PlayerPreviewComponent() {
+		super("Player Preview");
+	}
+	
+	@Override
+	public void render(int mouseX, int mouseY, float partialTicks) {
+		super.render(mouseX, mouseY, partialTicks);
+		
+		final Minecraft mc = Minecraft.getMinecraft();
+		EntityPlayer ent = mc.player;
+		
+		GlStateManager.pushMatrix();
+		GlStateManager.color(1, 1, 1);
+		RenderHelper.enableStandardItemLighting();
+		GlStateManager.enableAlpha();
+		GlStateManager.shadeModel(GL11.GL_FLAT);
+		GlStateManager.enableDepth();
+		
+		GlStateManager.enableColorMaterial();
+		GlStateManager.pushMatrix();
+		GlStateManager.translate(getX() + getW(), getY() + getH(), 50.0F);
```

Comment:
With the changes in size, this code won't work.  You want to place the player at the center bottom of the component so it renders upwards.